README.md

```
@@ -19,39 +19,77 @@ See [ROADMAP.md](./ROADMAP.md) for recent updates and upcoming features.
+#   REDIS_HOST=redis
+#   CELERY_BROKER_URL=redis://redis:6379/0
+#   CELERY_RESULT_BACKEND=redis://redis:6379/0
+#   MINIO_HOSTNAME=minio
+#   SMTP_HOST=mailpit
+#   OLLAMA_BASE_URL=http://ollama:11434/v1
+
+docker compose up -d
+```
+
+**Access:**
+- Frontend: http://localhost:3000
```

markdownlint-cli2 (0.20.0)

Bare URL used

(MD034, no-bare-urls)